### PR TITLE
Game page with grid and responsive

### LINKED ISSUE #375: Use grid for all elements on game page
ART to RT
Ant/Hive to Size

### PR DESCRIPTION
**Pull request by EgoriusRus**

### CODE REVIEW

**Review by kezlya — APPROVED** [review]
LGTM

File: components/AntHiveBotHorizontal.vue [hunk 1]
```diff
@@ -1,32 +1,74 @@
...
-     <img
+  <div class="chip">
+    <div class="information">
+      <div class="information-container">
+        <AntHiveButton
+          tile
+          color="accent"
+          @click="$emit('viewProfile', bot.displayName)"
+        >
+          View Profile
```
Comment (Klimonov) [inline components/AntHiveBotHorizontal.vue]: i18n

File: components/AntHiveBotHorizontal.vue [hunk 2]
```diff
@@ -83,19 +125,70 @@ export default {
...
+      font-weight: $font-weight-bold;
+    }
+  }
+  .information {
+    position: absolute;
+    min-width: 100%;
+    height: 100%;
+    transition: all 0.4s;
+    &:hover {
+      background: #040505b8;
```
Comment (Klimonov) [inline components/AntHiveBotHorizontal.vue]: remove to global, we will use it for enother components too

File: services/Game.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ async function getGames(params) {
 }
 
 async function getGame(id) {
-  const gameUrl = `/games/${id}`
+  const gameUrl = `/game/${id}`
```
Comment (Klimonov) [inline services/Game.js:8]: you need to use localPath for localization

File: components/AntHiveBotHorizontal.vue [hunk 4]
```diff
@@ -1,32 +1,74 @@
...
- <div class="chip">
-   <div class="img" :style="`background: center / cover no-repeat url(${getAvatar(bot.avatar, 200)})`">
-     <img
+  <div class="chip">
+    <div class="information">
+      <div class="information-container">
+        <AntHiveButton
+          tile
+          color="accent"
+          @click="$emit('viewProfile', bot.displayName)"
```
Comment (Klimonov) [inline components/AntHiveBotHorizontal.vue]: why not use viewProfile method in this component ?

File: components/AntHiveBotHorizontal.vue (hunk 2 above)
Comment (Klimonov) [reply components/AntHiveBotHorizontal.vue]: or you can use `$color-black-transparent: #000000d1;` from global.scss
I added it in my last PR for hover layouts

**Review by Klimonov — APPROVED** [review]
LGTM

**Pull request merged**